types/function_list_builder.go

```diff
@@ -51,19 +53,47 @@ func (s *FunctionLookupBuilder) Build() (map[string][]string, error) {
 		return serviceMap, marshalErr
 	}
 
+	topicDelim := ","
+
+	if delim, exists := os.LookupEnv("topic_delimiter"); exists && len(delim) > 0 {
+		topicDelim = delim
+	}
+
 	for _, function := range functions {
+
 		if function.Annotations != nil {
+
 			annotations := *function.Annotations
 
-			if topic, pass := annotations["topic"]; pass {
+			if topicNames, exist := annotations["topic"]; exist {
```

Comment:
Wondering if we should be moving `topic` > `topics` to outline that there can be multiple topics?